services/QuillLMS/lib/data/demo/activity_sessions.yml

```diff
@@ -0,0 +1,8487 @@
+---
```

Comment:
What is your thinking behind versioning these files, since they are intended to be clobbered?

Reply:
Answered above, but we will use these files to generate the demo accounts on any environment: local, staging, prod. They are intentionally in the repo since they rarely change. It's more consistent and faster to generate it from the static files.